
From: Hans Reiser <reiser@namesys.com>

From: Steven Cole <elenstev@mesatop.com>

Here are some trivial updates to the Changes file.  Please pass up the food
chain if OK.

Signed-off by: Steven Cole <elenstev@mesatop.com>

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Documentation/Changes |   12 ++++++++++++
 1 files changed, 12 insertions(+)

diff -puN Documentation/Changes~reiser4-doc-update Documentation/Changes
--- 25/Documentation/Changes~reiser4-doc-update	2004-08-21 23:55:55.878317776 -0700
+++ 25-akpm/Documentation/Changes	2004-08-21 23:55:55.882317168 -0700
@@ -56,6 +56,7 @@ o  module-init-tools      0.9.10        
 o  e2fsprogs              1.29                    # tune2fs
 o  jfsutils               1.1.3                   # fsck.jfs -V
 o  reiserfsprogs          3.6.3                   # reiserfsck -V 2>&1|grep reiserfsprogs
+o  reiser4progs           1.0.0                   # fsck.reiser4 -V
 o  xfsprogs               2.6.0                   # xfs_db -V
 o  pcmcia-cs              3.1.21                  # cardmgr -V
 o  quota-tools            3.09                    # quota -V
@@ -177,6 +178,13 @@ The reiserfsprogs package should be used
 versions of mkreiserfs, resize_reiserfs, debugreiserfs and
 reiserfsck. These utils work on both i386 and alpha platforms.
 
+Reiser4progs
+------------
+
+The reiser4progs package contains utilities for the reiser4 file system.
+Detailed instructions are provided in the README file located at:
+<ftp://ftp.namesys.com/pub/reiser4progs/README>.
+
 Xfsprogs
 --------
 
@@ -340,6 +348,10 @@ Reiserfsprogs
 -------------
 o  <http://www.namesys.com/pub/reiserfsprogs/reiserfsprogs-3.6.3.tar.gz>
 
+Reiser4progs
+------------
+o  <ftp://ftp.namesys.com/pub/reiser4progs/>
+
 Xfsprogs
 --------
 o  <ftp://oss.sgi.com/projects/xfs/download/>
_
